query-node/mappings/src/content/nft.ts

```diff
@@ -851,11 +851,11 @@ export async function contentNft_AuctionCanceled({ event, store }: EventContext
   await store.save<AuctionCanceledEvent>(announcingPeriodStartedEvent)
 }
 
-export async function contentNft_EnglishAuctionCompleted({ event, store }: EventContext & StoreContext): Promise<void> {
+export async function contentNft_EnglishAuctionSettled({ event, store }: EventContext & StoreContext): Promise<void> {
   // common event processing
 
   // memberId ignored here because it references member that called extrinsic - that can be anyone!
-  const [, /* memberId */ videoId] = new Content.EnglishAuctionCompletedEvent(event).params
+  const [winnerId, , videoId] = new Content.EnglishAuctionSettledEvent(event).params
```

Comment:
🤯 I had no idea you can skip vars like this, I always did `[value1, _, value2]`, cool trick!